File: src/main/java/raccoonlang/tokenizer/TokenNotRecognizedException.java
package raccoonlang.tokenizer;

public class TokenNotRecognizedException extends RuntimeException {
    public TokenNotRecognizedException(String filePath, String fileContent, int line, int column) {
        super(filePath + ":" + line + ":" + column + "|" + " Unknown token found near " + fileContent);
    }
}

File: Tokenizing/Exceptions/TokenNotRecognizedException.cs
namespace Raccoonlang.Tokenizing.Exceptions;

using System;

public class TokenNotRecognizedException : Exception
{
    public TokenNotRecognizedException(string filePath, int line, int column, string fileContent) 
    : base($"{filePath}:{line}:{column} | Unknown token found near {fileContent}!") {}
}